Lib/fontTools/ttLib/tables/_g_l_y_f.py

```diff
@@ -1070,11 +1070,12 @@ def compile(self, more, haveInstructions, glyfTable):
 				data = data + struct.pack(">HH", self.firstPt, self.secondPt)
 				flags = flags | ARG_1_AND_2_ARE_WORDS
 		else:
+			x, y = self.roundOffset()
```

Comment:
Unfortunately in many places we modify the font when writing it.  That's a bigger problem.

For this particular case I'd say it's simple enough that a method is not needed at al.